app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -277,32 +253,32 @@ def process_hardware(template_object, params, metadata, domain)
   def default_value(params, name)
     value = nil
     params.each do |param|
-      if param.name == name
-        value = param.value
+      if param[:name] == name
+        value = param[:value]
       end
     end
     value
   end
 
   def process_disks(hw_object, domain)
-    domain.devices.disks.each do |disk|
+    domain.dig(:devices, :disks).each do |disk|
```

Comment:
I think this might through an error if  domain.dig(:devices, :disks) is nil

Reply:
will add a check

Reply:
I do not see how it can be `nil` so there is no need to have additional check here.